install.sh

```diff
@@ -306,7 +306,7 @@ nvm_do_install() {
     command printf "${SOURCE_STR}"
   else
     case "${NVM_PROFILE-}" in
-      ".bashrc" | ".bash_profile" | ".zshrc")
+      *"/.bashrc" | *"/.bash_profile" | *"/.zshrc")
```

Comment:
Not sure if I understand it, looks like `test/install_script/nvm_do_install` is just for testing `nvm_do_install` function available?